[ADD] marketing automation: added 'getting started' section and docs … (#2552)
odoo/documentation: content/applications/marketing/marketing_automation/getting_started/target_audience.rst

icon beside them. The arrow signifies that the filter has more refined parameters within it. 

.. image:: target_audience/marketing-filters.png
   :align: center
   :alt: The drop-down filter menu in the Marketing Automation application.

Filters can be extended by adding *branches* and *nodes*. A *branch* creates a new line of
parameters, and further allows parameters to be clustered with *Any* or *All* statements, and a
*node* adds another parameter to the branch.

Every time a new branch is created, there are two options:

- Either the records can match :guilabel:`ALL` criteria for the upcoming rules (an AND statement
  where all criteria must match).

- Or, the records can match :guilabel:`ANY` criteria for the upcoming rules (an OR statement where
  only one of the criteria must match).

To change between these two options, simply click the drop-down arrow icon in the green box and
select :guilabel:`Any` or :guilabel:`All`.

To add a node, click on the :guilabel:`+ (plus sign icon)` and to add another branch click on the
:guilabel:`... (ellipses icon)`. To exclude a :guilabel:`node` (or a :guilabel:`branch`), click on
:guilabel:`x (delete)`.

> .. image:: target_audience/marketing-filter-nodes.png

Reviewer: Again the image is too wide and not taken in HD.
As well there is no differentiation in the filters so visually this is not as useful as an example that uses real any/all statement combinations.

I took a screenshot of a simple + common setup we use for our campaigns that shows the node/branch features as well as demonstrates a real world example. 
See here: https://i.imgur.com/2qC5ZTd.png (already cropped/compressed).

   :align: center
   :alt: The drop-down filter menu in the Marketing Automation application.

Use cases
=========

> The following scenarios are examples of some filters a marketing campaign might use.

Reviewer: Suggestion
```suggestion
The following scenarios below outline different combinations of filters a marketing campaign might
commonly use.
```

> **Scenario**: Narrow the target down to new opportunities in the pipeline.

Reviewer: Think this would better serve as an H3 to differentiate the scenarios and break up the text.
```suggestion
Scenario #1: Narrow the target down to new opportunities in the pipeline.
-------------------------------------------------------------------------
```

> While in Edit mode on a campaign template form, select the :guilabel:`Target` field, and click 

Reviewer: Remove trailing whitespace.
```suggestion
While in Edit mode on a campaign template form, select the :guilabel:`Target` field, and click
```

> :guilabel:`Search More` from the drop-down menu. Then, scroll down (or search for) 
> :guilabel:`Lead/Opportunity` and select it.

Reviewer: Suggestion to make instruction more concise.

```suggestion
:guilabel:`Search More` from the drop-down menu. Then, search for :guilabel:`Lead/Opportunity` and
select it.
```

> Next, click :guilabel:`Add Filter` in the :guilabel:`Filter` field. Click on the default 
> :guilabel:`ID` filter option in the first portion of the filter equation. Doing so reveals a 
> drop-down menu full of filter options. From this drop-down, scroll down (or search for) 
> :guilabel:`Type`.

Reviewer: Trailing whitespaces...
```suggestion
Next, click :guilabel:`Add Filter` in the :guilabel:`Filter` field. Click on the default
:guilabel:`ID` filter option in the first portion of the filter equation. Doing so reveals a
drop-down menu full of filter options. From this drop-down, scroll down (or search for)
:guilabel:`Type`.
```

Keep the second portion of the filter equation on the default :guilabel:`= (equal sign)`.

Then, change the third (and final) portion of the filter equation from :guilabel:`Lead` to 
:guilabel:`Opportunity`. The number of :guilabel:`Records` that fit this specific filter equation 
will change as the equation is customized.

Add another node to this filter by clicking the :guilabel:`+ (plus sign)` to the right of the 
equation.

> With "new" opportunities being the target of this filter, the second node will focus on only 
> locating opportunities that are in the :guilabel:`New` stage of the pipeline. To do that, select 
> the default :guilabel:`ID` from the first portion of the second filter equation, and scroll down 
> (or search for) :guilabel:`Stage` from the field drop-down menu.

Reviewer: Trailing whitespaces...
```suggestion
With "new" opportunities being the target of this filter, the second node will focus on only
locating opportunities that are in the :guilabel:`New` stage of the pipeline. To do that, select
the default :guilabel:`ID` from the first portion of the second filter equation, and scroll down
(or search for) :guilabel:`Stage` from the field drop-down menu.
```

Once again, leave the second portion of the filter equation on :guilabel:`= (equal sign)`.

Lastly, highlight the default value in the third (and final) portion of the second filter equation 
and type in :guilabel:`New`. With that completed, Odoo will only target opportunities that are in 
the "New" stage of the pipeline.

> .. image:: target_audience/filters-opportunities.png

Reviewer: Good screenshot, however doesn't need to be so wide.
Suggest replacing with this one: https://i.imgur.com/64Rh14D.png (already cropped + compressed).

   :align: center
   :alt: A standard scenario using filters in the Odoo Marketing Automation app.

> **Scenario**: Narrow the target down to the attendees of a specific event that purchased a specific
> category of ticket.

Reviewer: Condense and turn into H3
```suggestion
Scenario #2: Narrow the target down to the event attendees that purchased a specific ticket.
--------------------------------------------------------------------------------------------
```

> While in Edit mode on a campaign template form, select the :guilabel:`Target` field, and click 
> :guilabel:`Search More` from the drop-down menu. Then, scroll down (or search for) 
> :guilabel:`Event` and select it.

Reviewer: Trailing whitespaces...
```suggestion
While in Edit mode on a campaign template form, select the :guilabel:`Target` field, and click
:guilabel:`Search More` from the drop-down menu. Then, scroll down (or search for)
:guilabel:`Event` and select it.
```

> Next, click :guilabel:`Add Filter` in the :guilabel:`Filter` field. Click on the default 
> :guilabel:`ID` filter option in the first portion of the filter equation. Doing so reveals a 
> drop-down menu full of filter options. From this drop-down, scroll down (or search for) 
> :guilabel:`Event`.

Reviewer: Trailing whitespaces...
```suggestion
Next, click :guilabel:`Add Filter` in the :guilabel:`Filter` field. Click on the default
:guilabel:`ID` filter option in the first portion of the filter equation. Doing so reveals a
drop-down menu full of filter options. From this drop-down, scroll down (or search for)
:guilabel:`Event`.
```

Click the the default :guilabel:`= (equal sign)` in the second portion of the filter equation.
This reveals a drop-down menu. On this menu, select :guilabel:`contains`.

> In the third (and final) empty portion of the filter equation, type in the name of the event(s) 

Reviewer: ```suggestion
In the third (and final) empty portion of the filter equation, type in the name of the event(s)
```

that you would like Odoo to consider for this campaign filter.

> Add another node to this filter by clicking the :guilabel:`+ (plus sign)` to the right of the 

Reviewer: ```suggestion
Add another node to this filter by clicking the :guilabel:`+ (plus sign)` to the right of the
```

equation.

> The second node will focus on targeting this campaign to attendees who purchase a specific type of 
> ticket to the aforementioned event(s) in the first filter equation. To do that, select the default 
> :guilabel:`ID` from the first portion of the second filter equation, and scroll down (or search 
> for) :guilabel:`Event Ticket` from the field drop-down menu. Then, in that same drop-down menu, 
> select :guilabel:`Name`.

Reviewer: ```suggestion
The second node will focus on targeting this campaign to attendees who purchase a specific type of
ticket to the aforementioned event(s) in the first filter equation. To do that, select the default
:guilabel:`ID` from the first portion of the second filter equation, and scroll down (or search
for) :guilabel:`Event Ticket` from the field drop-down menu. Then, in that same drop-down menu,
select :guilabel:`Name`.
```

> Once again, click the the default :guilabel:`= (equal sign)` in the second portion of the filter 
> equation, and select :guilabel:`contains`.

Reviewer: ```suggestion
Once again, click the the default :guilabel:`= (equal sign)` in the second portion of the filter
equation, and select :guilabel:`contains`.
```

> Lastly, in the third (and final) portion of the second filter equation, which will be blank, type 
> in the name of the ticket type that should be used for the filter. In this case, 
> :guilabel:`Standard` is the name of the event ticket type for this sample filter.

Reviewer: ```suggestion
Lastly, in the third (and final) portion of the second filter equation, which will be blank, type
in the name of the ticket type that should be used for the filter. In this case,
:guilabel:`Standard` is the name of the event ticket type for this sample filter.
```

> .. image:: target_audience/filters-event-ticket.png

Reviewer: Too wide.
Replacement: https://i.imgur.com/smystFL.png

==================
Target an audience
==================

Getting marketing campaigns to the *right* audience is paramount when trying to grow a business.
Fortunately, the :guilabel:`Odoo Marketing Automation` application provides the right tools to take
marketing strategies to the next level.

Work with target filters
========================

When configuring the filters on a marketing campaign, there are some options that have an arrow
icon beside them, this signifies the parameter has more variables within it.

.. image:: target_audience/marketing-filters.png
   :align: center
   :alt: The drop-down filter menu in the Marketing Automation application.

> More than one parameter line can be configured. A :guilabel:`node (+)` can be added under what is
> called a :guilabel:`branch (…)`. To add a node, click on the :guilabel:`+` (plus sign icon) and to
> add another branch click on the :guilabel:`...` (ellipses icon).
> 
> To exclude a :guilabel:`node` (or a :guilabel:`branch`), click on :guilabel:`Delete node (x)`.
> 
> .. image:: target_audience/marketing-filter-nodes.png
>    :align: center
>    :alt: The drop-down filter menu in the Marketing Automation application.
> 
> Every time a new branch is created, there are two options, either the records can match
> :guilabel:`all` criteria for the upcoming rules, or the records can match :guilabel:`any` (meaning
> one rule or the other). To change between these two options, simply click the drop-down arrow icon 
> in the green box and select :guilabel:`Any` or :guilabel:`All`.

Reviewer: Suggestion to rewrite this entire section for more context in the writing and clarity. 
In order:
1. define the terms in hierarchical order (branch > node)
2. then provide usage instructions per term 
3. screenshot follows last (as _supplemental_ content, not the _main_ content that interrupts the writing)

```suggestion
Filters can be extended by adding *branches* and *nodes*. A *branch* creates a new line of
parameters, and further allows parameters to be clustered with *Any* or *All* statements, and a
*node* adds another parameter to the branch. 

Every time a new branch is created, there are two options: 

- either the records can match :guilabel:`ALL` criteria for the upcoming rules (an AND statement
where all criteria must match),
- or the records can match :guilabel:`ANY` criteria for the upcoming rules (an OR statement where
only one of the criteria must match). 

To change between these two options, simply click the drop-down arrow icon in the green box and
select :guilabel:`Any` or :guilabel:`All`.

To add a node, click on the :guilabel:`+ (plus sign icon)` and to add another branch click on the
:guilabel:`... (ellipses icon)`. To exclude a :guilabel:`node` (or a :guilabel:`branch`), click on
:guilabel:`x (delete)`.

.. image:: target_audience/marketing-filter-nodes.png
   :align: center
   :alt: The drop-down filter menu in the Marketing Automation application.
```

**Scenario**: Narrow the leads down to the ones that contain a certain tag name.

.. image:: target_audience/filters-tag-name.png
   :align: center
   :alt: An example of what a tag name filter looks like in Odoo Marketing Automation.

**Scenario**: Narrow the target down to the attendees of a specific event that purchased a specific
category of ticket.

.. image:: target_audience/filters-event-ticket.png
   :align: center
   :alt: How an event ticket filter appears in the Odoo Marketing Automation application.

**Scenario**: Narrow the target down to the employees who have kids.

.. image:: target_audience/filters-kids.png
   :align: center
   :alt: A filter based on the number of children an employee has in their family.

**Scenario**: Narrow the target down to records with a :guilabel:`Helpdesk` ticket solved.

.. image:: target_audience/filters-helpdesk-ticket.png
   :align: center
   :alt: A standard Helpdesk ticket related filter in Odoo Marketing Automation.

Favorite Filters
================

Custom filters can be saved for future use in Odoo in just one click. 

Once the custom filter has been configured, there is an option to save it, by clicking the

> :guilabel:`Disc Drive` icon next to the :guilabel:`Upload a favorite filter` field.

Author: You're right! I believe Favorite Filters didn't come into effect until v15. How should I proceed - just delete it entirely?

> :guilabel:`Disc Drive` icon next to the :guilabel:`Upload a favorite filter` field.

Author: I went ahead and deleted it because it wouldn't apply to v14. Thanks again for catching this!

.. image:: target_audience/favorite-filter-field.png
   :align: center
   :alt: Favorite filter field on template.

When clicked, a tiny pop-up will appear. 

Here, a title can be added to the :guilabel:`Favorite Filter`. Then click :guilabel:`Add` to save

> the filter. An overview of all Favorite Filters is accessible under  :guilabel:`Configuration` >

Author: No, this was my fault. It only applies to v15 and on. I deleted this section entirely to avoid confusion. Sorry for the inconvenience.

> the filter. An overview of all Favorite Filters is accessible under  :guilabel:`Configuration` >

Reviewer 2: No problem! Save this info and when this PR is closed and merged, we can open a new PR targeting v15.0 to add in this information 😸
